.github/workflows/sanitizer.yml

```
@@ -38,15 +38,15 @@ jobs:
     name: undefined-behavior-sanitizer
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
     - uses: lukka/get-cmake@latest    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)